Evaluation/Workflow.m

```diff
@@ -0,0 +1,78 @@
+% Script to perform a Qualification Plan workflow
+% Qualification Plan Workflow developed with Matlab 2017b
+% --------------------------------------------------------------
+
+close all
+tic
+
+if exist(fullfile(cd,'re_input'),'dir')>0 rmdir(fullfile(cd,'re_input'),'s'); end
```

Comment:
using `cd` will not work in _compiled_ version of the Reporting Engine.

Reply:
we will update workflow.m in aunified fashion during release phase!